concepticondata/conceptlists/Carling-2017-209b.tsv

```diff
@@ -1,210 +1,210 @@
+Carling-2017-209b-14	14	2633	BLACK OR DARK	black (colour)	2014		
+Carling-2017-209b-15	15	946	BLOOD	blood (n)	2015		
+Carling-2017-209b-16	16	175	BLOW (OF WIND)	to blow	2016		
+Carling-2017-209b-17	17	1394	BONE	bone	2017		
+Carling-2017-209b-18	18	1402	BREAST	breast	2018		
+Carling-2017-209b-19	19	1407	BREATHE	to breathe	2019		
+Carling-2017-209b-20	20	1428	BURNING	to burn (intr.)	2020		
+Carling-2017-209b-21	21	2099	CHILD	child	2021		
+Carling-2017-209b-22	22	1489	CLOUD	cloud (not fog)	2022		
+Carling-2017-209b-23	23	2483	COLD (OF WEATHER)	cold (of weather)	2023		
+Carling-2017-209b-24	24	1446	COME	to come	2024		
+Carling-2017-209b-25	25	1420	COUNT	to count	2025		
+Carling-2017-209b-26	26	1432	CUT	to cut	2026		
+Carling-2017-209b-27	27	1225	DAY (NOT NIGHT)	day	2027		
+Carling-2017-209b-28	28	1494	DIE	to die	2028		
+Carling-2017-209b-29	29	1418	DIG	to dig	2029		
+Carling-2017-209b-30	30	1230	DIRTY	dirty	2030		
+Carling-2017-209b-31	31	2009	DOG	dog	2031		
+Carling-2017-209b-32	32	1401	DRINK	to drink	2032		
+Carling-2017-209b-33	33	1398	DRY	dry (adj)	2033		
+Carling-2017-209b-34	34	379	BLUNT	dull	2034		
+Carling-2017-209b-35	35	2	DUST	dust	2035		
+Carling-2017-209b-36	36	1247	EAR	ear	2036		
+Carling-2017-209b-37	37	1228	EARTH (SOIL)	earth (soil, ground)	2037		
+Carling-2017-209b-38	38	1336	EAT	to eat	2038		
+Carling-2017-209b-39	39	744	EGG	egg	2039		
+Carling-2017-209b-40	40	1248	EYE	eye	2040		
+Carling-2017-209b-41	41	1280	FALL	to fall	2041		
+Carling-2017-209b-42	42	1406	FAR	far	2042		
+Carling-2017-209b-43	43	323	FAT (ORGANIC SUBSTANCE)	fat (organic substance)	2043		
+Carling-2017-209b-44	44	1217	FATHER	father	2044		
+Carling-2017-209b-45	45	1419	FEAR (BE AFRAID)	to fear	2045		
+Carling-2017-209b-46	46	1201	FEATHER	feather (not down)	2046		
+Carling-2017-209b-47	47	1242	FEW	few	2047		
+Carling-2017-209b-48	48	1423	FIGHT	to fight	2048		
+Carling-2017-209b-49	49	1258	FINGERNAIL	fingernail	2049		
+Carling-2017-209b-50	50	221	FIRE	fire	2050		
+Carling-2017-209b-51	51	227	FISH	fish (n)	2051		
+Carling-2017-209b-52	52	493	FIVE	five	2052		
+Carling-2017-209b-53	53	1574	FLOAT	to float	2053		
+Carling-2017-209b-54	54	2003	FLOW	to flow	2054		
+Carling-2017-209b-55	55	239	FLOWER	flower	2055		
+Carling-2017-209b-56	56	1441	FLY (MOVE THROUGH AIR)	to fly	2056		
+Carling-2017-209b-57	57	249	FOG	fog (mist)	2057		
+Carling-2017-209b-58	58	1301	FOOT	foot (not leg)	2058		
+Carling-2017-209b-59	59	1500	FOUR	four	2059		
+Carling-2017-209b-60	60	1431	FREEZE	to freeze	2060		
+Carling-2017-209b-61	61	1507	FRUIT	fruit	2061		
+Carling-2017-209b-62	62	1429	FULL	full	2062		
+Carling-2017-209b-63	63	1447	GIVE	to give	2063		
+Carling-2017-209b-64	64	1035	GOOD	good	2064		
+Carling-2017-209b-65	65	606	GRASS	grass	2065		
+Carling-2017-209b-66	66	1425	GREEN	green (colour)	2066		
+Carling-2017-209b-67	67	1334	GUTS	guts	2067		
+Carling-2017-209b-68	68	2648	HAIR (HEAD)	hair (of head)	2068		
+Carling-2017-209b-69	69	1277	HAND	hand (not arm)	2069		
```

Comment:
Hihi, I always say this, that people write elicitation glosses like "hand (not arm)", but never actually found this in the literature!!!